src/doc_builder/utils.py

```diff
@@ -74,3 +76,95 @@ def get_doc_config():
     Returns the `doc_config` if it has been loaded.
     """
     return doc_config
+
+
+def convert_parametype_numpydoc_to_groupsdoc(paramtype):
+    """
+    Converts parameter type str that is written in numpystyle to groupsstyle (e.g. transformers docs).
+
+    Args:
+    paramtype (`str`): The name of the object to document.
+
+    Example:
+    ```py
+    >>> s = "bool, optional, default True"
+    >>> convert_numpydoc_to_rstdoc(s)
+    '`bool`, `optional`, defaults to `True`'
+    ```
+    """
+    paramtype = re.sub(r"\s*\(.*\)$", "", paramtype).replace("default", "defaults to")
+    arr = paramtype.split(", ")
+    for i, s in enumerate(arr):
+        s = s.split(" ")
+        if "optional" in s[-1]:
+            # parameter is optional or not
+            s[-1] = f"`{s[-1]}`"
+        elif "defaults" in s:
+            # parameter default value
+            valtype = type(eval(s[-1]))
+            non_obj_types = [str, int, float]
+            s[-1] = s[-1] if valtype in non_obj_types else f":obj:`{s[-1]}`"
```

Comment:
We actually use backticks for the strings :-)
```suggestion
            non_obj_types = [int, float]
            s[-1] = s[-1] if valtype in non_obj_types else f":obj:`{s[-1]}`"
```

Reply:
handled